plugins/metric_links/src/Metric.php

```diff
@@ -224,17 +210,18 @@ public function getStatusHelpText($machine_name)
 
     /**
      * Determine if a status is an error and should be logged
-     * 
-     * @param LinkStatus $status
+     *
+     * @param Page\Link $link
+     * @internal param LinkStatus $status
      * @return bool
      */
-    public function isError(LinkStatus $status)
+    public function isError(Page\Link $link)
     {
-        if (in_array($status->http_code, $this->options['http_error_codes'])) {
+        if (in_array($link->original_status_code, $this->options['http_error_codes'])) {
             return true;
         }
 
-        if ($status->curl_code && empty($status->http_code)) {
+        if ($link->original_curl_code && empty($link->http_code)) {
```

Comment:
fixed in https://github.com/mfairchild365/site_master/commit/2fcb56003297d6134681df1b89dcc1b9e78efc48